{
  try {
    if (!exceptionThrownLatch.await(timeout,TimeUnit.MILLISECONDS)) {
      fail("There was no exception notification");
    }
  }
 catch (  InterruptedException e) {
    throw new RuntimeException(e);
  }
  return this;
}
